native/src/feature/onboarding/screens/SendPhoneVerificationCode.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { View, Text, Button } from "react-native";
+import { useNavigation } from "@react-navigation/native";
+
+export const SendPhoneVerificationCode: React.FC<{}> = () => {
+  const navigation = useNavigation();
+
+  return (
+    <>
+      <View>
+        <Text>SendPhoneVerificationCode</Text>
+        <Button
+          title={`Next`}
```

Comment:
No es necesario utilizar `` cuando no haces _string interpolation_: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

```suggestion
          title="Next"
```

Reply:
- He atendido tu comentario.